src/printer/tir_text_printer.cc

```diff
@@ -465,27 +465,30 @@ Doc TIRTextPrinter::VisitStmt_(const EvaluateNode* op) {
   return doc;
 }
 
-inline const char* ForType2String(ForType t) {
+inline const char* ForKind2String(ForKind t) {
```

Comment:
Consider move this to tir/stmt.h?

Reply:
This case this is specifically used in text printer, so we plan to keep it here for now

Reply:
I thought our convention is do define a stringify method immediately after defining an enum class...